UPGRADE.md

```diff
@@ -10,9 +10,27 @@ before finalizing the upgrade process.
 These changes have not yet been released and this area's purpose is to keep
```

Comment:
I think we can probably remove this file as it no longer serves a purpose (I don't think anyone is on version 0.3.0)